iotdb/src/main/java/org/apache/iotdb/db/utils/OpenFileNumUtil.java

```diff
@@ -125,10 +125,10 @@ private static int getPid() {
         in1.close();
         pro1.destroy();
       } catch (IOException e) {
-        LOGGER.error("Cannot get pid of IoTDB process because {}", e.getMessage());
+        LOGGER.error("Cannot get PID of IoTDB process because {}", e.getMessage());
```

Comment:
Suggest using  'LOGGER.error("Cannot get PID of IoTDB process because ", e);'

Reply:
changed